manager/assets/modext/widgets/element/modx.tree.element.js

```diff
@@ -451,11 +451,15 @@ Ext.extend(MODx.tree.Element,MODx.tree.Tree,{
 
     ,_getCategoryMenu: function(n) {
         var a = n.attributes;
+        var name = a.text;
+        if (a.data.category && a.data.elementCount > 0) {
+            var name = a.data.category + ' [' + a.data.elementCount + ']';
+        }
```

Comment:
I do not understand the question. PR changes brackets for nested category elements. Also in this block.

Reply:
I understand what the PR does, but I can't seem to see any difference when I remove this block and revert line 458/462 to use just `a.text`

Reply:
I see it in the right click menu and also in the tree with square brackets even without this block.

Reply:
Yes you are right. I first corrected the script in the tree, and then the processors (because the categories themselves and the categories of elements are slightly different entities) and did not notice that in `a.text` the value was already passed with the new brackets. I'll fix it now.